traad/rope/findit.py

```diff
@@ -22,7 +37,7 @@ def _find_locations(self, func, path, offset):
             self.root_project,
             self.get_resource(path),
             offset)
-        return emap(location_to_tuple, results)
+        return list(map(location_to_dict, results))
```

Comment:
`list(map(...))` is exactly what `emap` does. Honestly, I'm happy to dump eagertools - it was really just a little experiment of mine - but I thought I'd mention that in case you wanted to stick with it.

Reply:
Sure thing - I should explain overtly. I replaced it as I had a little trouble loading the dependency and this was somewhat more overt.